Samkiroko/youtube-clone
# video and sidebar  setup

**What does this pr do**
this is the main video column and the sidebar

**How should this be manually tested**
cd to the project open the index.html you will see the main video column and the sidebar
